### DISCUSSION TITLE
FXIOS-1360 ⁃ I can't build using Xcode 12.0

### DISCUSSION THREAD

**Discussion by data-sync-user** [mozilla-mobile/firefox-ios#7931] [General]
I am followed the steps. But, I can't get successful build. Seems, I am missing soothing  that's not listed in **Building the code**.

I tried last release **firefox~~ios~~30.0** as well. Latest one and last release builds with exactly same error.

**Build input file cannot be found**

https://ibb.co/3NQQvQP

So, I deleted the unknown files in each target and builded again then I got below error,

**Type 'GleanMetrics' has no member 'ReadingList'**

https://ibb.co/TT86mvr

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1360)

**Comment by dnarcese** [comment]
Can you try deleting the `Carthage` folder and then running `./bootstrap.sh --force`

**Comment by Koalageddonn** [comment]
I am unable to build using Xcode 12.0. Has anyone else encountered this issue and found a solution?
